recipes/folly/all/conanfile.py

```diff
@@ -107,11 +108,18 @@ def source(self):
         extracted_dir = self.name + "-" + self.version
         os.rename(extracted_dir, self._source_subfolder)
```

Comment:
Ideal I'd suggestion to use `strip_root` but I can not comment on the right lines

```suggestion
        extracted_dir = self.name + "-" + self.version
        tools.rename(extracted_dir, self._source_subfolder)
```

Reply:
I don't understand how `strip_root` might help. It would move all the content to the root of the build dir making it harder to work with. Will change `os` to `tools` per your suggestion.